ArduCopter/motors.pde

```diff
@@ -152,30 +158,6 @@ static bool init_arm_motors(bool arming_from_gcs)
     }
     calc_distance_and_bearing();
 
-    uint32_t gyro_cal_time = 0;
-    if(did_ground_start == false) {
-        uint32_t gyro_cal_begin = millis();
-
-        startup_ground(true);
```

Comment:
We might want to call this with a false argument when the checks pass. I am not totally sure that the ahrs init functions are needed but we prob want to set the land flags to true (set_land_complete and set_land_complete_maybe).